romancal/regtest/test_wfi_pipeline.py

```diff
@@ -0,0 +1,144 @@
+                  str(model.meta.cal_step.saturation == 'COMPLETE'))
+    assert model.meta.cal_step.saturation == 'COMPLETE'
+
+    # DMS87 data quality tests
+    pipeline.log.info('DMS87 MSG: Testing existence of data quality array (dq) in'
+                  'Level 2 image output.......' +
+                  str("dq" in model.keys()))
+    assert "dq" in model.keys()
+    pipeline.log.info('DMS87 MSG: Testing existence of general error array (err) in'
+                  'Level 2 image output.......' +
+                  str("err" in model.keys()))
+    assert "err" in model.keys()
+    pipeline.log.info('DMS87 MSG: Testing existence of Poisson noise variance array (var_poisson) in'
+                  'Level 2 image output.......' +
+                  str("var_poisson" in model.keys()))
+    assert "var_poisson" in model.keys()
+    pipeline.log.info('DMS87 MSG: Testing existence of read noise variance array (var_rnoise) in'
+                  'Level 2 image output.......' +
+                  str("var_rnoise" in model.keys()))
+    assert "var_rnoise" in model.keys()
+    pipeline.log.info('DMS87 MSG: Testing existence of flatfield uncertainty variance array (var_flat) in'
+                  'Level 2 image output.......' +
+                  str("var_flat" in model.keys()))
+    assert "var_flat" in model.keys()
+
+    # DMS88 total exposure time test
+    pipeline.log.info('DMS88 MSG: Testing existence of total exposure time (exposure_time)'
+                  'in Level 2 image output.......' +
+                  str("exposure_time" in model.meta.exposure))
+    assert "exposure_time" in model.meta.exposure
+
+    # DMS89 WCS tests
+    pipeline.log.info('DMS89 MSG: Testing that the wcs bounding'
+                  'box was generated.......' +
+                  str(((len(model.meta.wcs.bounding_box) == 2) and
+                        (type(model.meta.wcs.bounding_box[0][0]) == float))))
+    assert ((len(model.meta.wcs.bounding_box) == 2) and
+            (type(model.meta.wcs.bounding_box[0][0]) == float))
+
+    # Save original wcs information
+    orig_wcs = copy.deepcopy(model.meta.wcs)
+    del model.meta['wcs']
+
+    # Create new pointing for the model
+    # RA & Dec are each shifted + 10 degrees, unless they are near the upper limit,
+    # in which case they are shifted -10 degrees.
+    if model.meta.wcsinfo.ra_ref < 350.0:
+        model.meta.wcsinfo.ra_ref += 10.0
+    else:
+        model.meta.wcsinfo.ra_ref -= 10.0
+
+    if model.meta.wcsinfo.dec_ref < 80.0:
+        model.meta.wcsinfo.dec_ref += 10.0
+    else:
+        model.meta.wcsinfo.dec_ref -= 10.0
+
+    # Create new wcs object for the new pointing
+    model = AssignWcsStep.call(model)
+
+    pipeline.log.info('DMS89 MSG: Testing that the different pointings create differing wcs.......' +
+                  str(model.meta.wcs.footprint != orig_wcs.footprint))
+    assert model.meta.wcs.footprint != orig_wcs.footprint
```

Comment:
`footprint` is a function, not a property, so you need to call it.
It returns a  numpy array, so perhaps a test here can be checking that the difference in the footprint is ~10 deg, something like

```
footprint1 = model.meta.wcs.footprint()
footprint2 = orig_wcs.footprint()

assert_allclose(np.abs(footprint1 - footprint2), 10)
```
You will need to specify an appropriate `atol`  difference to the assert call.

Reply:
I changed it to a test of RA & DEC of a center pixel (2048, 2048).